### DISCUSSION TITLE
Working on Ventura?

### DISCUSSION THREAD

**Discussion by dhcmrl** [kvic-z/SsdPmEnabler#26] [Q&A]
Hi,

First of all, I just want to thank you for your work on this.

I'm attempting to install the kext on my Macbook Pro 13" 2017 (non-touchbar) running Ventura 13.6. I'm getting an error in terminal when I attempt to move SsdPmEnabler.kext from Downloads into /Library/Extensions. `unable to copy extended attributes to /Library/Extensions/SsdPmEnabler.kext: Operation not permitted`

Is there any reason this wouldn't work on Ventura? Or am I potentially missing something else?

**Comment by theshawnz** [comment]
This sounds like you are trying to install with SIP enabled? Did you double check this? I was able to get it installed on Ventura though no positive result with regards to my SSD consuming less energy. I think it's due to some dependencies that are not present in Ventura. 


<img width="451" alt="Screenshot 2024-03-31 at 12 09 44 AM" src="https://github.com/kvic-z/SsdPmEnabler/assets/40644028/14e782bf-033c-420f-9019-1b999fd2245d">
